apps/src/gamelab/GameLabP5.js

```diff
@@ -451,13 +452,40 @@ GameLabP5.prototype.registerP5EventHandler = function (eventName, handler) {
   this.p5[eventName] = handler;
 };
 
+GameLabP5.prototype.setP5FrameRate = function () {
+  if (!this.p5) {
+    return;
+  }
+  if (this.stepSpeed < 1) {
+    // TODO: properly handle overriding frameRate
```

Comment:
Can you clarify what you mean with this TODO?

Reply:
👍 
Updating the comment to read:
```
      // TODO: properly handle overriding frameRate (this implementation doesn't
      // account for any calls to frameRate() that occur while we are in the
      // slow mode - we'll need to patch p5 to capture those and update
      // this.prevFrameRate)
```